lib/read.rb

```diff
@@ -1,2 +1,3 @@
 class Read
+
```

Comment:
You should avoid checking in unnecessary changes like this as they are clutter to anyone reviewing